src/config.c

```diff
@@ -56,6 +56,8 @@ void Config_init(void)
     /*--------------- Cache related ---------------*/
     CONFIG.cache_enabled = 0;
 
+    CONFIG.cache_clear = 0;
```

Comment:
You don't need this.

Reply:
https://github.com/fangfufu/httpdirfs/commit/002d65f3a5eb76c035bc0d593ec2c9dc3a0eb4c6

Also removes member added to struct